assets/maps/random_rooms/3x3/nope_no_room_70.dmm

```diff
@@ -0,0 +1,28 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/simulated/wall/auto/reinforced,
+/area/dmm_suite/clear_area)
+"G" = (
+/obj/decal/cleanable/writing{
+	words = "FUCK YOU!!!";
+	desc = "Someone's scribbled something here in furious, block-letter text.";
+	uses_default_material_name = s
```

Comment:
yeah uhh

Reply:
Can't tell what this comment is about, or what changes are being requested :/. First time I've had a change req on a mapping PR, still newish

Reply:
you were editing the writing and you mistyped `s`

Reply:
Oh, thanks, will fix